sys/include/suit/v4/policy.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2019 Koen Zandberg
+ *               2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2019 Inria
+ *               2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup     sys_suit_v4
+ * @brief       SUIT policy definitions
+ *
+ * @{
+ *
+ * @brief       SUIT policy definitions
+ * @author      Koen Zandberg <koen@bergzand.net>
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ */
+
+#ifndef SUIT_V4_POLICY_H
+#define SUIT_V4_POLICY_H
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "uuid.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief bitfield of required policies
+ * @{
+ */
+#define SUIT_VALIDATED_AUTH         0x1
```

Comment:
Can you add descriptions for these defines, e.g. `/**< seq_nr <= other imag */`. Also mark as  unsupported or something like tthat for `SUIT_VALIDATED_AUTH`.

Reply:
done